multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/validators/parameters/IdleRoutesValidator.java

```diff
@@ -0,0 +1,24 @@
+package org.cloudfoundry.multiapps.controller.core.validators.parameters;
+
+import java.util.Map;
+
+import org.cloudfoundry.multiapps.controller.core.model.SupportedParameters;
+
+public class IdleRoutesValidator extends RoutesValidator {
+
+    public IdleRoutesValidator(String namespace, boolean applyNamespaceGlobal) {
```

Comment:
Is this constructor even needed now?

Reply:
Yes, because it is necessary to initialize parent constructor